runtime/commonMain/src/kotlinx/serialization/builtins/BuiltinsMigrations.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.builtins
+
+import kotlinx.serialization.*
+import kotlinx.serialization.internal.*
+
+@Deprecated(
+    message = "Deprecated in the favour of Unit.serializer() extension",
+    level = DeprecationLevel.ERROR,
+    replaceWith = ReplaceWith("Unit.serializer()", imports = ["kotlinx.serialization.builtins.serializer"])
+)
+public fun UnitSerializer(): KSerializer<Unit> = Unit.serializer()
+
+@Deprecated(
+    message = "Deprecated during serialization 1.0 API stabilization",
+    level = DeprecationLevel.ERROR,
+    replaceWith = ReplaceWith("ListSerializer(this)")
+)
+public val <T> KSerializer<T>.list: KSerializer<List<T>>
```

Comment:
We can leave this function internal since it's heavily used in our tests

Reply:
I'd prefer to use the same code style we recommend to our users